blog/2020-09-14-debezium-auto-create-topics.adoc

```diff
@@ -0,0 +1,235 @@
+= Auto-creating Debezium Change Data Topics
+rkerner
+:awestruct-tags: [ kafka, topics, production, news ]
```

Comment:
what do you think about tags @gunnarmorling ?

Reply:
LGTM. Might also add "discussion" which we used for these kinds of more generic posts.

Reply:
done